Godeps/Godeps.json

```diff
@@ -8,8 +8,8 @@
 	"Deps": [
 		{
 			"ImportPath": "github.com/Azure/azure-sdk-for-go/storage",
-			"Comment": "v1.2-334-g95361a2",
-			"Rev": "95361a2573b1fa92a00c5fc2707a80308483c6f9"
+			"Comment": "v6.0.0-beta",
```

Comment:
I don't like using a package marked as beta into a production system.

Is there something between v1.2 and v6 that is more stable?